assets/component-facets.css

```diff
@@ -272,11 +272,40 @@
   padding: 0.5rem 2rem;
```

Comment:
You're looking at the wrong "Color" filter, remember you had two set up? Since you've searched for "bag", that doesn't match the product you attached the visual filter to, so it won't show up.

You could search for "*" instead and you should see it.